docs/user-manuals/fine-grained-cpu-orchestration.md

```diff
@@ -0,0 +1,138 @@
+
+To improve the performance of CPU-sensitive workloads, koord-scheduler provides a mechanism of fine-grained CPU
+orchestration. It enhances the CPU management of Kubernetes and supports detailed NUMA-locality and CPU exclusions.
+
+For more information, please see [Design: Fine-grained CPU orchestration](/docs/designs/fine-grained-cpu-orchestration).
+
+## Setup
+
+### Prerequisite
+
+- Kubernetes >= 1.18
+- Koordinator >= 0.5
+
+### Installation
+
+Please make sure Koordinator components are correctly installed in your cluster. If not, please refer to [Installation](/docs/installation).
+
+### Configurations
+
+You can enable load-aware scheduling and configure the rules by modifying the ConfigMap `koord-scheduler-config` in the helm chart.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: koord-scheduler-config
+  ...
+data:
+  koord-scheduler-config: |
+    apiVersion: kubescheduler.config.k8s.io/v1beta2
+    kind: KubeSchedulerConfiguration
+    profiles:
+      - schedulerName: koord-scheduler
+      - pluginConfig:
+        - name: NodeNUMAResource
+          args:
+            apiVersion: kubescheduler.config.k8s.io/v1beta2
+            kind: NodeNUMAResource
+            # the CPU binding policy ('None', 'FullPCPUs', 'SpreadByPCPUs', 'ConstrainedBurst')
+            preferredCPUBindPolicy: FullPCPUs
+            # the NUMA allocation strategy ('MostAllocated', 'LeastAllocated', 'DistributeEvenly')
+            numaAllocateStrategy: MostAllocated
+            # the scoring strategy
+            scoringStrategy:
+              # the kind of scoring strategy ('MostAllocated', 'BalancedAllocation', 'LeastAllocated')
+              type: MostAllocated
+              # the weights of each resource type
+              resources:
+              - name: cpu
+                weight: 2
+              - name: memory
+                weight: 1
+        plugins:
+          # enable the NodeNUMAResource plugin
+          preFilter:
+            enabled:
+              - name: NodeNUMAResource
+          filter:
+            enabled:
+              - name: NodeNUMAResource
+              ...
+          score:
+            enabled:
+              - name: NodeNUMAResource
+                weight: 1
+              ...
+          reserve:
+            enabled:
+              - name: NodeNUMAResource
+          preBind:
+            enabled:
+              - name: NodeNUMAResource
+```
+
+The koord-scheduler takes this ConfigMap as [scheduler Configuration](https://kubernetes.io/docs/reference/scheduling/config/).
+New configurations will take effect after the koord-scheduler restarts.
+
+## Use Fine-grained CPU Orchestration
+
+1. Create an `nginx` deployment with the YAML file below. Set QoS classes of pods as [`LSE`](/docs/architecture/qos#definition) or [`LSR`](/docs/architecture/qos#definition).
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-lsr
+  labels:
+    app: nginx-lsr
+    koordinator.sh/qosClass: LSR
```

Comment:
This label is useless and easy to misunderstand

Reply:
removed